### PR TITLE
Respect cleanup table's "group" column in PCA vis

### PR DESCRIPTION
**Pull request by waxlamp**

### CODE REVIEW

**Comment by waxlamp** [comment]
@subdavis, I made the requested change; please take a look and let me know what you think.

File: web/src/components/vis/VisPca.vue [hunk 1]
```diff
@@ -94,6 +94,26 @@ export default {
...
     },
+    dataset: {
+      required: true,
+      validator: prop => typeof prop === 'object'
+        && 'source' in prop
+        && typeof prop.source === 'object'
+        && 'columns' in prop.source
+        && Array.isArray(prop.source.columns),
+    },
+    id: {
```
Comment (subdavis) [inline web/src/components/vis/VisPca.vue]: Can be removed?

File: web/src/components/Transform.vue [hunk 2]
```diff
@@ -116,5 +116,5 @@ v-layout.transform-view(row, fill-height)
   v-layout(v-else, row, fill-height, ref="contentarea")
     .pa-4
       h3.headline.ml-5 Principal Component Analysis
-      vis-pca(:width="800", :height="600", :raw-points="pcaData")
+      vis-pca(:width="800", :height="600", :raw-points="pcaData", :dataset="dataset" :id="id")
```
Comment (subdavis) [inline web/src/components/Transform.vue]:
```suggestion
      vis-pca(:width="800", :height="600", :raw-points="pcaData", :dataset="dataset")
```

**Comment by jbeezley** [comment]
Should be fixed now.

**Pull request merged**